### PR TITLE
Fix #3617 minor bug fixes and improvements

### LINKED ISSUE #3617: Highlight GetFeatureInfo results on map
### Issues
 - depends on #3616

### Acceptance criteria
- A toggle button should be present in the top toolbar of the featureinfo button, called "highlight features"
- The button visibility should be configurable
- When the highlight features button is activated, the clicked geometry should be highlighted on map, shown together with the marker, on feature info click.
- The geometry should be highlighted only if the following conditions are satisfied (Otherwise only the marker will be on map) : 
    - The server supports GFI JSON format
    -  The layer is a vector layer and so the GFI can return also the geometry
- If more then one layer has feature info response, only the geometries shown also in the GFI panel will be visible (so only one layer at the same time). 

Preview ( [mockup](https://demo.geo-solutions.it/mockups/mapstore2/annotation_styles/#/viewer/openlayers/new) ): 
image

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
This PR fixes some minor issues on highlight feature info feature functionality: 
 - [ ] Moved navigation buttons from tool bar on the right side, to prevent appearing/disappearing buttons to shift.
 - [ ] Allow configurability (using localConfig) of highlight color
-  [ ] Fixed german translations

## Issues
 - Fix #3617 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Bugfix

**What is the current behavior?** (You can also link to an open issue here)
- Toolbar makes hard to navigate features, when more than one layer is present
- Tooltip for german translation is inccorrect  "Stoppen Sie die Hervorhebung von Funktionen" 

**What is the new behavior?**
- Navigation and tools are separated. Navigation buttons, when present, are only disabled on first/last page (before they was not visible).
- Highlight color can be configured using localConfig.json (documented how to in mapinfo reducer). 
- Tooltip above replaced with : "Stoppen Sie die Markierung von Features".

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

### CODE REVIEW

File: web/client/components/data/identify/IdentifyContainer.jsx [hunk 1]
```diff
@@ -106,14 +109,22 @@ module.exports = props => {
                         onChangeFormat={onChangeFormat}
                     /> || null,
                     <GeocodeViewer latlng={latlng} revGeocodeDisplayName={revGeocodeDisplayName} {...props}/>,
-                    buttons.length > 0 ? (
```
Comment (MV88) [inline web/client/components/data/identify/IdentifyContainer.jsx]: you sure about this? if i remember correctly it was messing up the layout (try it with IE11)

File: web/client/components/data/identify/IdentifyContainer.jsx [hunk 2]
```diff
@@ -76,10 +78,10 @@ module.exports = props => {
         lngCorrected = lngCorrected - 360 * Math.floor(lngCorrected / 360 + 0.5);
     }
     const Viewer = viewer;
-    const buttons = getButtons({...props, lngCorrected, validResponses, latlng});
+    // TODO: put all the header (Toolbar, navigation, coordinate editor) outside the container
+    const buttons = getToolButtons({...props, lngCorrected, validResponses, latlng});
```
Comment (MV88) [inline web/client/components/data/identify/IdentifyContainer.jsx]: let's call it toolsButtons or buttonsTools

File: web/client/selectors/mapInfo.js [hunk 3]
```diff
@@ -6,7 +6,7 @@
 * LICENSE file in the root directory of this source tree.
 */
 
-const { get, isArray } = require('lodash');
+const { get, omit, isArray } = require('lodash');
```
Comment (MV88) [inline web/client/selectors/mapInfo.js]: they say that pluck is to be preferred to omit for performance reason.
(if you know precisely which props to pass. i mean)

File: web/client/components/data/identify/IdentifyContainer.jsx (hunk 2 above)
Comment (offtherailz) [reply web/client/components/data/identify/IdentifyContainer.jsx]: :ok:

File: web/client/selectors/mapInfo.js (hunk 3 above)
Comment (offtherailz) [reply web/client/selectors/mapInfo.js]: We should prefer readability over premature optimization

File: web/client/components/data/identify/IdentifyContainer.jsx (hunk 1 above)
Comment (offtherailz) [reply web/client/components/data/identify/IdentifyContainer.jsx]: Checked, buttons array is never empty, so this check dosn't make sense

**Pull request merged**